src/troll/trollResponses.ts

```diff
@@ -0,0 +1,29 @@
+export const customTrollResponses = new Map<string, string[]>();
+
+// Griefa
+customTrollResponses.set('300457650615746561', [
+  "Mate, you're about as useful as a chocolate teapot in the Outback.",
+  "You're as reliable as a waterproof teabag, mate.",
+  "You're like a painting by a blindfolded gorilla.",
+  "You smell like a wet koala, mate.",
+  "Hey.",
+  "Can you let your mum know that I'm not coming over tonight?",
+  "If you wanted me to touch you, you could have just asked.",
+  "pssst...I know your secrets.",
+]);
+
+// Andy
+customTrollResponses.set('243440367066677251', [
+  "psst...No one uses NDCore.",
+  "ND = Andy? So is it actually AndyCore? Who is names a framework after themselves?",
```

Comment:
"Who is names a framework..."

small typo

Reply:
fixed